### PR TITLE
Edit css animations animation timing function #3529

### LINKED ISSUE #3529: [Edit] CSS [Edit] CSS Animations: animation-timing-function
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

We would like an edit a term within the CSS Animations entry. The entry lives in docs/content/css/concepts/animations/terms/animation-timing-function/animation-timing-function.md

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

Please:
1) Add supplemental media (preferred .png or .gif, <1MB).
2) Media (images, gifs) should accurately represent the existing code blocks in the example sections and fit in-line.

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by Brag2gr8**
### Description

Edited the Animation-Timing-Function in CSS Animations and added a media example.
Entry location docs/content/css/concepts/animations/terms/animation-timing-function/animation-timing-function.md
Media location doc/media/css-animation-timing-function.gif . Closes #3529 

### Type of Change

Editing an existing entry.
Adding a media

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by Brag2gr8** [comment]
@SSwiniarski

File: content/css/concepts/animations/terms/animation-timing-function/animation-timing-function.md [hunk 1]
```diff
@@ -59,3 +61,6 @@ div {
   }
 }
 ```
+
+**Note**
```
Comment (dakshdeepHERE) [inline content/css/concepts/animations/terms/animation-timing-function/animation-timing-function.md:65]:
```suggestion
```
No need for this note.

File: content/css/concepts/animations/terms/animation-timing-function/animation-timing-function.md [hunk 2]
```diff
@@ -59,3 +61,6 @@ div {
...
+![css-animation-timing-function-example](docs/media/css-animation-timing-function.gif)
```
Comment (dakshdeepHERE) [inline content/css/concepts/animations/terms/animation-timing-function/animation-timing-function.md:66]: Make the description better for this gif

File: content/css/concepts/animations/terms/animation-timing-function/animation-timing-function.md [hunk 3]
```diff
@@ -12,38 +12,40 @@ CatalogContent:
...
+- `ease-out`: Easing curve for an animation that starts quickly and decelerates.
+- `ease-in-out`: Easing curve for an animation that starts and ends smoothly.
+- `cubic-bezier()`: Easing curve defined using cubic Bezier functions.
 
 **Note:** Applied property by property from keyframe to keyframe.
 
 ## Example 1
 
-Apply `ease-in-out` to the `animation-timing-function` property of the `div` element:
+Applying `ease-in-out` to the `animation-timing-function` property of a div element:
```
Comment (dakshdeepHERE) [inline content/css/concepts/animations/terms/animation-timing-function/animation-timing-function.md:38]:
```suggestion
Applying `ease-in-out` to the `animation-timing-function` property of a `div` element:
```

**Comment by Brag2gr8** [comment]
@dakshdeepHERE please check the changes out

**Review by dakshdeepHERE — APPROVED** [review]
Looks good for second review 🚀

**Review by SSwiniarski — APPROVED** [review]
@Brag2gr8 Looks good.

**Pull request merged**